diff --git a/node_modules/hexo-douban/.eslintrc b/node_modules/hexo-douban/.eslintrc
index 73019a1..4537cda 100644
--- a/node_modules/hexo-douban/.eslintrc
+++ b/node_modules/hexo-douban/.eslintrc
@@ -1,6 +1,9 @@
 {
   "extends": "hexo",
   "root": true,
+  "rules": {
+    "no-console": 1
+  },
   "env": {
     "browser": true
   }
diff --git a/node_modules/hexo-douban/index.js b/node_modules/hexo-douban/index.js
index 7ff7966..73df50b 100644
--- a/node_modules/hexo-douban/index.js
+++ b/node_modules/hexo-douban/index.js
@@ -55,7 +55,7 @@ hexo.extend.console.register('douban', 'Generate pages from douban', options, fu
   }
 
   var doubanLoadingPath = '/assets/douban-loading.gif';
-  //Load static files
+  // Load static files
   hexo.extend.generator.register('douban-gif', function (locals) {
     return {
       path: doubanLoadingPath,
@@ -65,15 +65,15 @@ hexo.extend.console.register('douban', 'Generate pages from douban', options, fu
     };
   });
 
-  //Register route
+  // Register route
   names.forEach(name => {
     hexo.extend.generator.register(name, require('./lib/' + name + '-generator'));
   })
 
   var self = this;
-  var publicDir = self.public_dir;
+  var publicDir = this.public_dir;
 
-  //Generate files
+  // Generate files
   self.load().then(function () {
     if(!fs.existsSync(publicDir)){
       fs.mkdirSync(publicDir);
@@ -81,9 +81,12 @@ hexo.extend.console.register('douban', 'Generate pages from douban', options, fu
     if(!fs.existsSync(path.join(publicDir,'assets'))){
       fs.mkdirSync(path.join(publicDir,'assets'));
     }
-    self.route.get(doubanLoadingPath)._data().then(function (stream) {
-      stream.pipe(fs.createWriteStream(path.join(publicDir, doubanLoadingPath)));
-    });
+    self.route.get(doubanLoadingPath);
+    // .data().then(function (stream) {
+    //   stream.pipe(
+    //     fs.createWriteStream(path.join(publicDir, doubanLoadingPath))
+    //     );
+    // });
     names.forEach(name => {
       var id = name + "/index.html";
       self.route.get(id) && self.route.get(id)._data().then(function (contents) {
diff --git a/node_modules/hexo-douban/lib/templates/book.ejs b/node_modules/hexo-douban/lib/templates/book.ejs
index 943070e..496b614 100644
--- a/node_modules/hexo-douban/lib/templates/book.ejs
+++ b/node_modules/hexo-douban/lib/templates/book.ejs
@@ -1,5 +1,5 @@
 <blockquote>
-    <p><%- quote; %></p>
+    <p class="note note-primary"><%- quote; %></p>
 </blockquote>
 
 <style>
@@ -47,4 +47,3 @@
     <% include index.js %>
     <% include pagination.js %>
 </script>
-
diff --git a/node_modules/hexo-douban/lib/templates/bookRead.ejs b/node_modules/hexo-douban/lib/templates/bookRead.ejs
index 88ce77d..bb54921 100644
--- a/node_modules/hexo-douban/lib/templates/bookRead.ejs
+++ b/node_modules/hexo-douban/lib/templates/bookRead.ejs
@@ -1,5 +1,5 @@
 <div class="hexo-douban-item">
-    <div class="hexo-douban-picture"><img src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
+    <div class="hexo-douban-picture"><img class="shadowed" src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
     <div class="hexo-douban-info">
         <div class="hexo-douban-title"><a target="_blank" href="<%= item.alt %>"> <%= item.title %></a></div>
         <div class="hexo-douban-meta"><%= item.pub %></div>
diff --git a/node_modules/hexo-douban/lib/templates/bookReading.ejs b/node_modules/hexo-douban/lib/templates/bookReading.ejs
index 4ba3e14..f188567 100644
--- a/node_modules/hexo-douban/lib/templates/bookReading.ejs
+++ b/node_modules/hexo-douban/lib/templates/bookReading.ejs
@@ -1,8 +1,8 @@
 <div class="hexo-douban-item">
-    <div class="hexo-douban-picture"><img src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
+    <div class="hexo-douban-picture"><img class="shadowed" src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
     <div class="hexo-douban-info">
         <div class="hexo-douban-title"><a target="_blank" href="<%= item.alt %>"> <%= item.title %></a></div>
-        <div class="hexo-douban-meta"><%= item.pub %>  </div>
+        <div class="hexo-douban-meta"><%= item.pub %></div>
         <div class="hexo-douban-meta"><%= item.updated %> / <%= item.tags %></div>
         <div class="hexo-douban-comments"><%= item.comment %></div>
     </div>
diff --git a/node_modules/hexo-douban/lib/templates/bookWish.ejs b/node_modules/hexo-douban/lib/templates/bookWish.ejs
index 631e759..f188567 100644
--- a/node_modules/hexo-douban/lib/templates/bookWish.ejs
+++ b/node_modules/hexo-douban/lib/templates/bookWish.ejs
@@ -1,9 +1,8 @@
 <div class="hexo-douban-item">
-    <div class="hexo-douban-picture"><img src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
+    <div class="hexo-douban-picture"><img class="shadowed" src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
     <div class="hexo-douban-info">
         <div class="hexo-douban-title"><a target="_blank" href="<%= item.alt %>"> <%= item.title %></a></div>
-        <div class="hexo-douban-meta"><%= item.pub %>
-        </div>
+        <div class="hexo-douban-meta"><%= item.pub %></div>
         <div class="hexo-douban-meta"><%= item.updated %> / <%= item.tags %></div>
         <div class="hexo-douban-comments"><%= item.comment %></div>
     </div>
diff --git a/node_modules/hexo-douban/lib/templates/game.ejs b/node_modules/hexo-douban/lib/templates/game.ejs
index e927f3a..7600bf9 100644
--- a/node_modules/hexo-douban/lib/templates/game.ejs
+++ b/node_modules/hexo-douban/lib/templates/game.ejs
@@ -1,5 +1,5 @@
 <blockquote>
-    <p><%- quote %></p>
+    <p class="note note-primary"><%- quote %></p>
 </blockquote>
 
 <style>
@@ -47,4 +47,3 @@
     <% include index.js %>
     <% include pagination.js %>
 </script>
-
diff --git a/node_modules/hexo-douban/lib/templates/gamePlayed.ejs b/node_modules/hexo-douban/lib/templates/gamePlayed.ejs
index feba4f2..7562333 100644
--- a/node_modules/hexo-douban/lib/templates/gamePlayed.ejs
+++ b/node_modules/hexo-douban/lib/templates/gamePlayed.ejs
@@ -1,11 +1,11 @@
 <div class="hexo-douban-item">
-    <div class="hexo-douban-picture"><img src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
+    <div class="hexo-douban-picture"><img class="shadowed" src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
     <div class="hexo-douban-info">
         <div class="hexo-douban-title">
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
-        <div class="hexo-douban-meta"><%= item.date %> / <%= item.tags %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-meta"><%= item.info %> </div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
         <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
diff --git a/node_modules/hexo-douban/lib/templates/gamePlaying.ejs b/node_modules/hexo-douban/lib/templates/gamePlaying.ejs
index 05a525f..7562333 100644
--- a/node_modules/hexo-douban/lib/templates/gamePlaying.ejs
+++ b/node_modules/hexo-douban/lib/templates/gamePlaying.ejs
@@ -1,12 +1,11 @@
 <div class="hexo-douban-item">
-    <div class="hexo-douban-picture"><img src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
+    <div class="hexo-douban-picture"><img class="shadowed" src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
     <div class="hexo-douban-info">
         <div class="hexo-douban-title">
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
         <div class="hexo-douban-meta"><%= item.info %> </div>
-        <div class="hexo-douban-meta"><%= item.date %> / <%= item.tags %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
         <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
diff --git a/node_modules/hexo-douban/lib/templates/gameWish.ejs b/node_modules/hexo-douban/lib/templates/gameWish.ejs
index ce61c0a..0a5309d 100644
--- a/node_modules/hexo-douban/lib/templates/gameWish.ejs
+++ b/node_modules/hexo-douban/lib/templates/gameWish.ejs
@@ -1,10 +1,11 @@
 <div class="hexo-douban-item">
-    <div class="hexo-douban-picture"><img src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
+    <div class="hexo-douban-picture"><img class="shadowed" src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
     <div class="hexo-douban-info">
         <div class="hexo-douban-title">
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
         <div class="hexo-douban-meta"><%= item.info %></div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
diff --git a/node_modules/hexo-douban/lib/templates/index.css b/node_modules/hexo-douban/lib/templates/index.css
index 7ea3a5a..998d440 100644
--- a/node_modules/hexo-douban/lib/templates/index.css
+++ b/node_modules/hexo-douban/lib/templates/index.css
@@ -9,11 +9,13 @@
 }
 
 .hexo-douban-active {
-    background: #657b83;
+    background: #6d6985;
     color: #fff;
 }
 
 .hexo-douban-item {
+    display: flex;
+    align-items: center;
     padding-bottom: 10px;
     position: relative;
     clear: both;
@@ -29,21 +31,38 @@
 }
 
 .hexo-douban-picture {
+    display: flex;
+    align-items: center;
     position: absolute;
     left: 0;
-    top: 10px;
-    width: 100px;
+    max-width: 100px;
+    height: 150px;
+}
+
+img.shadowed {
+    display: block;
+    box-shadow: 0 5px 11px 0 rgba(0, 0, 0, 0.18), 0 4px 15px 0 rgba(0, 0, 0, 0.15);
+    max-width: 100%;
+    max-height: 100%;
 }
 
 .hexo-douban-info {
     padding-left: 120px;
 }
 
+.hexo-douban-title {
+    font-size: 16px;
+}
+
 .hexo-douban-meta {
-    font-size: 12px;
+    font-size: 14px;
     padding-right: 10px;
 }
 
+.hexo-douban-description {
+    font-size: 10px;
+}
+
 .hexo-douban-comments {
     font-size: 12px;
 }
@@ -59,7 +78,7 @@
 }
 
 .hexo-douban-button:hover {
-    background: #657b83;
+    background: #6d6985;
     color: #fff;
 }
 
@@ -70,4 +89,3 @@
 .hexo-douban-show {
     display: block;
 }
-
diff --git a/node_modules/hexo-douban/lib/templates/movie.ejs b/node_modules/hexo-douban/lib/templates/movie.ejs
index dec866a..b86387e 100644
--- a/node_modules/hexo-douban/lib/templates/movie.ejs
+++ b/node_modules/hexo-douban/lib/templates/movie.ejs
@@ -1,5 +1,5 @@
 <blockquote>
-    <p><%- quote; %></p>
+    <p class="note note-primary"><%- quote; %></p>
 </blockquote>
 
 <style>
@@ -45,4 +45,3 @@
     <% include index.js %>
     <% include pagination.js %>
 </script>
-
diff --git a/node_modules/hexo-douban/lib/templates/movieWatched.ejs b/node_modules/hexo-douban/lib/templates/movieWatched.ejs
index feba4f2..34ca50d 100644
--- a/node_modules/hexo-douban/lib/templates/movieWatched.ejs
+++ b/node_modules/hexo-douban/lib/templates/movieWatched.ejs
@@ -1,11 +1,10 @@
 <div class="hexo-douban-item">
-    <div class="hexo-douban-picture"><img src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
+    <div class="hexo-douban-picture"><img class="shadowed" src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
     <div class="hexo-douban-info">
         <div class="hexo-douban-title">
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
-        <div class="hexo-douban-meta"><%= item.date %> / <%= item.tags %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
         <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
diff --git a/node_modules/hexo-douban/lib/templates/movieWatching.ejs b/node_modules/hexo-douban/lib/templates/movieWatching.ejs
index ce61c0a..2526e6d 100644
--- a/node_modules/hexo-douban/lib/templates/movieWatching.ejs
+++ b/node_modules/hexo-douban/lib/templates/movieWatching.ejs
@@ -1,10 +1,11 @@
 <div class="hexo-douban-item">
-    <div class="hexo-douban-picture"><img src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
+    <div class="hexo-douban-picture"><img class="shadowed" src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
     <div class="hexo-douban-info">
         <div class="hexo-douban-title">
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
-        <div class="hexo-douban-meta"><%= item.info %></div>
+        <div class="hexo-douban-description"><%= item.info %></div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
diff --git a/node_modules/hexo-douban/lib/templates/movieWish.ejs b/node_modules/hexo-douban/lib/templates/movieWish.ejs
index ce61c0a..2526e6d 100644
--- a/node_modules/hexo-douban/lib/templates/movieWish.ejs
+++ b/node_modules/hexo-douban/lib/templates/movieWish.ejs
@@ -1,10 +1,11 @@
 <div class="hexo-douban-item">
-    <div class="hexo-douban-picture"><img src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
+    <div class="hexo-douban-picture"><img class="shadowed" src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
     <div class="hexo-douban-info">
         <div class="hexo-douban-title">
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
-        <div class="hexo-douban-meta"><%= item.info %></div>
+        <div class="hexo-douban-description"><%= item.info %></div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
