diff --git a/node_modules/hexo-douban/.eslintrc b/node_modules/hexo-douban/.eslintrc
index 73019a1..4537cda 100644
--- a/node_modules/hexo-douban/.eslintrc
+++ b/node_modules/hexo-douban/.eslintrc
@@ -1,6 +1,9 @@
 {
   "extends": "hexo",
   "root": true,
+  "rules": {
+    "no-console": 1
+  },
   "env": {
     "browser": true
   }
diff --git a/node_modules/hexo-douban/index.js b/node_modules/hexo-douban/index.js
index 7ff7966..73df50b 100644
--- a/node_modules/hexo-douban/index.js
+++ b/node_modules/hexo-douban/index.js
@@ -55,7 +55,7 @@ hexo.extend.console.register('douban', 'Generate pages from douban', options, fu
   }
 
   var doubanLoadingPath = '/assets/douban-loading.gif';
-  //Load static files
+  // Load static files
   hexo.extend.generator.register('douban-gif', function (locals) {
     return {
       path: doubanLoadingPath,
@@ -65,15 +65,15 @@ hexo.extend.console.register('douban', 'Generate pages from douban', options, fu
     };
   });
 
-  //Register route
+  // Register route
   names.forEach(name => {
     hexo.extend.generator.register(name, require('./lib/' + name + '-generator'));
   })
 
   var self = this;
-  var publicDir = self.public_dir;
+  var publicDir = this.public_dir;
 
-  //Generate files
+  // Generate files
   self.load().then(function () {
     if(!fs.existsSync(publicDir)){
       fs.mkdirSync(publicDir);
@@ -81,9 +81,12 @@ hexo.extend.console.register('douban', 'Generate pages from douban', options, fu
     if(!fs.existsSync(path.join(publicDir,'assets'))){
       fs.mkdirSync(path.join(publicDir,'assets'));
     }
-    self.route.get(doubanLoadingPath)._data().then(function (stream) {
-      stream.pipe(fs.createWriteStream(path.join(publicDir, doubanLoadingPath)));
-    });
+    self.route.get(doubanLoadingPath);
+    // .data().then(function (stream) {
+    //   stream.pipe(
+    //     fs.createWriteStream(path.join(publicDir, doubanLoadingPath))
+    //     );
+    // });
     names.forEach(name => {
       var id = name + "/index.html";
       self.route.get(id) && self.route.get(id)._data().then(function (contents) {
diff --git a/node_modules/hexo-douban/lib/templates/bookReading.ejs b/node_modules/hexo-douban/lib/templates/bookReading.ejs
index 4ba3e14..c85e4a7 100644
--- a/node_modules/hexo-douban/lib/templates/bookReading.ejs
+++ b/node_modules/hexo-douban/lib/templates/bookReading.ejs
@@ -2,7 +2,7 @@
     <div class="hexo-douban-picture"><img src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
     <div class="hexo-douban-info">
         <div class="hexo-douban-title"><a target="_blank" href="<%= item.alt %>"> <%= item.title %></a></div>
-        <div class="hexo-douban-meta"><%= item.pub %>  </div>
+        <div class="hexo-douban-meta"><%= item.pub %></div>
         <div class="hexo-douban-meta"><%= item.updated %> / <%= item.tags %></div>
         <div class="hexo-douban-comments"><%= item.comment %></div>
     </div>
diff --git a/node_modules/hexo-douban/lib/templates/bookWish.ejs b/node_modules/hexo-douban/lib/templates/bookWish.ejs
index 631e759..c85e4a7 100644
--- a/node_modules/hexo-douban/lib/templates/bookWish.ejs
+++ b/node_modules/hexo-douban/lib/templates/bookWish.ejs
@@ -2,8 +2,7 @@
     <div class="hexo-douban-picture"><img src="<%= root %>/assets/douban-loading.gif" data-src="<%= item.image %>" referrerPolicy="no-referrer" /></div>
     <div class="hexo-douban-info">
         <div class="hexo-douban-title"><a target="_blank" href="<%= item.alt %>"> <%= item.title %></a></div>
-        <div class="hexo-douban-meta"><%= item.pub %>
-        </div>
+        <div class="hexo-douban-meta"><%= item.pub %></div>
         <div class="hexo-douban-meta"><%= item.updated %> / <%= item.tags %></div>
         <div class="hexo-douban-comments"><%= item.comment %></div>
     </div>
diff --git a/node_modules/hexo-douban/lib/templates/gamePlayed.ejs b/node_modules/hexo-douban/lib/templates/gamePlayed.ejs
index feba4f2..31549c3 100644
--- a/node_modules/hexo-douban/lib/templates/gamePlayed.ejs
+++ b/node_modules/hexo-douban/lib/templates/gamePlayed.ejs
@@ -4,8 +4,8 @@
         <div class="hexo-douban-title">
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
-        <div class="hexo-douban-meta"><%= item.date %> / <%= item.tags %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-meta"><%= item.info %> </div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
         <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
diff --git a/node_modules/hexo-douban/lib/templates/gamePlaying.ejs b/node_modules/hexo-douban/lib/templates/gamePlaying.ejs
index 05a525f..31549c3 100644
--- a/node_modules/hexo-douban/lib/templates/gamePlaying.ejs
+++ b/node_modules/hexo-douban/lib/templates/gamePlaying.ejs
@@ -5,8 +5,7 @@
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
         <div class="hexo-douban-meta"><%= item.info %> </div>
-        <div class="hexo-douban-meta"><%= item.date %> / <%= item.tags %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
         <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
diff --git a/node_modules/hexo-douban/lib/templates/gameWish.ejs b/node_modules/hexo-douban/lib/templates/gameWish.ejs
index ce61c0a..26c4df1 100644
--- a/node_modules/hexo-douban/lib/templates/gameWish.ejs
+++ b/node_modules/hexo-douban/lib/templates/gameWish.ejs
@@ -5,6 +5,7 @@
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
         <div class="hexo-douban-meta"><%= item.info %></div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
diff --git a/node_modules/hexo-douban/lib/templates/index.css b/node_modules/hexo-douban/lib/templates/index.css
index 7ea3a5a..26d138b 100644
--- a/node_modules/hexo-douban/lib/templates/index.css
+++ b/node_modules/hexo-douban/lib/templates/index.css
@@ -14,6 +14,8 @@
 }
 
 .hexo-douban-item {
+    display: flex;
+    align-items: center;
     padding-bottom: 10px;
     position: relative;
     clear: both;
@@ -29,21 +31,38 @@
 }
 
 .hexo-douban-picture {
+    display: flex;
+    align-items: center;
     position: absolute;
     left: 0;
-    top: 10px;
-    width: 100px;
+    max-width: 100px;
+    height: 150px;
+}
+
+img {
+    display: block;
+    box-shadow: 0 5px 11px 0 rgba(0, 0, 0, 0.18), 0 4px 15px 0 rgba(0, 0, 0, 0.15);
+    max-width: 100%;
+    max-height: 100%;
 }
 
 .hexo-douban-info {
     padding-left: 120px;
 }
 
+.hexo-douban-title {
+    font-size: 16px;
+}
+
 .hexo-douban-meta {
-    font-size: 12px;
+    font-size: 14px;
     padding-right: 10px;
 }
 
+.hexo-douban-description {
+    font-size: 10px;
+}
+
 .hexo-douban-comments {
     font-size: 12px;
 }
@@ -70,4 +89,3 @@
 .hexo-douban-show {
     display: block;
 }
-
diff --git a/node_modules/hexo-douban/lib/templates/movieWatched.ejs b/node_modules/hexo-douban/lib/templates/movieWatched.ejs
index feba4f2..995a69e 100644
--- a/node_modules/hexo-douban/lib/templates/movieWatched.ejs
+++ b/node_modules/hexo-douban/lib/templates/movieWatched.ejs
@@ -4,8 +4,7 @@
         <div class="hexo-douban-title">
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
-        <div class="hexo-douban-meta"><%= item.date %> / <%= item.tags %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
         <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
diff --git a/node_modules/hexo-douban/lib/templates/movieWatching.ejs b/node_modules/hexo-douban/lib/templates/movieWatching.ejs
index ce61c0a..f86e126 100644
--- a/node_modules/hexo-douban/lib/templates/movieWatching.ejs
+++ b/node_modules/hexo-douban/lib/templates/movieWatching.ejs
@@ -4,7 +4,8 @@
         <div class="hexo-douban-title">
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
-        <div class="hexo-douban-meta"><%= item.info %></div>
+        <div class="hexo-douban-description"><%= item.info %></div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
diff --git a/node_modules/hexo-douban/lib/templates/movieWish.ejs b/node_modules/hexo-douban/lib/templates/movieWish.ejs
index ce61c0a..f86e126 100644
--- a/node_modules/hexo-douban/lib/templates/movieWish.ejs
+++ b/node_modules/hexo-douban/lib/templates/movieWish.ejs
@@ -4,7 +4,8 @@
         <div class="hexo-douban-title">
             <a target="_blank" href="<%= item.alt %>"><%= item.title %></a>
         </div>
-        <div class="hexo-douban-meta"><%= item.info %></div>
+        <div class="hexo-douban-description"><%= item.info %></div>
+        <div class="hexo-douban-meta"><%= item.date %> / <%= item.recommend %> </div>
+        <div class="hexo-douban-comments"> <%= item.comment %></div>
     </div>
 </div>
-
