diff --git a/node_modules/hexo-douban/.eslintrc b/node_modules/hexo-douban/.eslintrc
index 73019a1..4537cda 100644
--- a/node_modules/hexo-douban/.eslintrc
+++ b/node_modules/hexo-douban/.eslintrc
@@ -1,6 +1,9 @@
 {
   "extends": "hexo",
   "root": true,
+  "rules": {
+    "no-console": 1
+  },
   "env": {
     "browser": true
   }
diff --git a/node_modules/hexo-douban/index.js b/node_modules/hexo-douban/index.js
index 7ff7966..73df50b 100644
--- a/node_modules/hexo-douban/index.js
+++ b/node_modules/hexo-douban/index.js
@@ -55,7 +55,7 @@ hexo.extend.console.register('douban', 'Generate pages from douban', options, fu
   }
 
   var doubanLoadingPath = '/assets/douban-loading.gif';
-  //Load static files
+  // Load static files
   hexo.extend.generator.register('douban-gif', function (locals) {
     return {
       path: doubanLoadingPath,
@@ -65,15 +65,15 @@ hexo.extend.console.register('douban', 'Generate pages from douban', options, fu
     };
   });
 
-  //Register route
+  // Register route
   names.forEach(name => {
     hexo.extend.generator.register(name, require('./lib/' + name + '-generator'));
   })
 
   var self = this;
-  var publicDir = self.public_dir;
+  var publicDir = this.public_dir;
 
-  //Generate files
+  // Generate files
   self.load().then(function () {
     if(!fs.existsSync(publicDir)){
       fs.mkdirSync(publicDir);
@@ -81,9 +81,12 @@ hexo.extend.console.register('douban', 'Generate pages from douban', options, fu
     if(!fs.existsSync(path.join(publicDir,'assets'))){
       fs.mkdirSync(path.join(publicDir,'assets'));
     }
-    self.route.get(doubanLoadingPath)._data().then(function (stream) {
-      stream.pipe(fs.createWriteStream(path.join(publicDir, doubanLoadingPath)));
-    });
+    self.route.get(doubanLoadingPath);
+    // .data().then(function (stream) {
+    //   stream.pipe(
+    //     fs.createWriteStream(path.join(publicDir, doubanLoadingPath))
+    //     );
+    // });
     names.forEach(name => {
       var id = name + "/index.html";
       self.route.get(id) && self.route.get(id)._data().then(function (contents) {
diff --git a/node_modules/hexo-douban/lib/templates/index.css b/node_modules/hexo-douban/lib/templates/index.css
index 7ea3a5a..a3bdc47 100644
--- a/node_modules/hexo-douban/lib/templates/index.css
+++ b/node_modules/hexo-douban/lib/templates/index.css
@@ -14,6 +14,8 @@
 }
 
 .hexo-douban-item {
+    display: flex;
+    align-items: center;
     padding-bottom: 10px;
     position: relative;
     clear: both;
@@ -29,10 +31,19 @@
 }
 
 .hexo-douban-picture {
+    display: flex;
+    align-items: center;
     position: absolute;
     left: 0;
-    top: 10px;
-    width: 100px;
+    max-width: 100px;
+    height: 150px;
+}
+
+img {
+    display: block;
+    box-shadow: 0 5px 11px 0 rgba(0, 0, 0, 0.18), 0 4px 15px 0 rgba(0, 0, 0, 0.15);
+    max-width: 100%;
+    max-height: 100%;
 }
 
 .hexo-douban-info {
@@ -40,7 +51,7 @@
 }
 
 .hexo-douban-meta {
-    font-size: 12px;
+    font-size: 16px;
     padding-right: 10px;
 }
 
@@ -70,4 +81,3 @@
 .hexo-douban-show {
     display: block;
 }
-
